src/_includes/cloud/error-codes.md

```diff
@@ -147,6 +147,8 @@ Warning errors indicate a problem with the Magento Commerce Cloud project config
 | 2023 | install-update:split-db | Enabling a split database will be skipped. |  |
 | 2024 | install-update:split-db | The SPLIT_DB variable is missing the configuration for split connection types. |  |
 | 2025 | install-update:split-db | Slave connection not set. |  |
+| 2026 | pre-deploy:restore-writable-dirs | Failed to restore some data generated during the build phase to the mounted directories | Check the `cloud.log` for more information. |
+| 2027 | validate-config:mage-mode-variable | Environment variable MAGE_MODE has value with not supported mode  | Remove MAGE_MODE environment variable or change its value to "production". Magento Cloud supports only "production" mode. |
```

Comment:
Suggest changing the error code description for 2027 in the code to:

The mode value specified in the MAGE_MODE environment variable is not supported.

Reply:
Updated